initramfs/scripts/bootloader-script

```diff
@@ -0,0 +1,190 @@
+# -*- shell-script -*- vim:ft=sh:
```

Comment:
Just out of curiosity - how is this script triggered? via kernel cmdline "BOOT=bootloader-script" ? Or some other mechanism?